Modules/Microsoft365DSC/DSCResources/MSFT_EXOMailboxAuditBypassAssociation/MSFT_EXOMailboxAuditBypassAssociation.schema.mof

```diff
@@ -0,0 +1,13 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXOMailboxAuditBypassAssociation")]
+class MSFT_EXOMailboxAuditBypassAssociation : OMI_BaseResource
+{
+    [Key, Description("The Identity parameter specifies the user account or computer account where you want to view the value of the AuditBypassEnabled property.")] String Identity;
+    [Write, Description("The AuditBypassEnabled parameter specifies whether audit bypass is enabled for the user or computer.")] Boolean AuditBypassEnabled;
+    [Write, Description("Specifies if this instance should exist."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
```

Comment:
We should remove this parameter since the resource appears to be a tenant wide setting.

Reply:
removed it.